documentation/docs/libraries/nodejs/how_to/_run_how_tos.mdx

```diff
@@ -6,8 +6,8 @@ the library.
 ## Set Up Your `.env` File
```

Comment:
Just in case, we've also updated these instructions on https://github.com/iotaledger/wallet.rs/pull/1917 to include the `npm i` from the examples directory.

Reply:
Maybe we can wait until #1917 is merged and then I'll rebase this branch :relaxed: